plugins/lsp/lsp-packages.nix

```diff
@@ -358,6 +352,7 @@
     rescriptls = "rescript-language-server";
     # This is not entirely true, but the server is deprecated
     rls = "rustup";
+    roslyn_ls = "roslyn-ls";
```

Comment:
Need to disable testing this package. It's been broken for a while.

Reply:
Hm, actually we're not seeing the same "marked insecure" issue here, we actually attempt to build, but fail while building a transitive:

```
dotnet-vmr-10.0.0-rc.1.drv' failed with exit code 139;
./prep-source-build.sh: line 363:  6041 Segmentation fault         (core dumped)
```

Reply:
No this has an issue on nixpkgs specifically for it already. https://github.com/NixOS/nixpkgs/issues/431811